python_src/src/lamp_py/tableau/pipeline.py

```diff
@@ -27,7 +27,7 @@ def create_hyper_jobs() -> List[HyperJob]:
         HyperStaticFeedInfo(),
         HyperStaticRoutes(),
         HyperStaticStops(),
-        HyperStaticStopTimes(),
+        # HyperStaticStopTimes(),
```

Comment:
Don't run `static_stop_times` write until we figure out a more memory efficient solution